backend/development.ini.sample

```diff
@@ -857,6 +857,11 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 # to connect to a cluster, it is possible to provided multiple hosts/ports as a comma separated list
 ; search.elasticsearch.host = localhost,
 ; search.elasticsearch.port = 9200,
+# elasticsearch security configuration
+; search.elasticsearch.ssl = False
+; search.elasticsearch.ssl.ca_certs =
+; search.elasticsearch.auth.username =
+; search.elasticsearch.auth.password =
```

Comment:
Can we put this 5 lines just before example ?

I think `auth` can be naming `basic_auth`
What kind of file do we need to put in `search.elasticsearch.ssl.ca_certs` ? Path with file or just filename ?

Can you add also an example in example area, to present which kind of value are accepted ?

Reply:
maybe replace `search.elasticsearch.ssl` by `search.elasticsearch.ssl.activated` (like we have for some parameters with value True or False).

Is it possible to use `search.elasticsearch.ssl=False` (or true) without auth username and password ?

Reply:
What is going on if I have a cluster of elasticsearch ? Each node receive same auth or just one in the list of node ?

Reply:
> Is it possible to use search.elasticsearch.ssl=False (or true) without auth username and password ?

It's required: these are two separate features:
- use basic_auth or no auth
- use a SSL based connection

@floreal15 please add a thumb up if it is working like this.

Also I like the renaming proposals `ssl.activated` and `basic_auth` instead of `auth`

For the certs file path, we should have something explicit about the expected content of the file. Is it a SSL chain (cert + intermediates ?) a cert only? Also giving an example with a full path should be the right way to help users

Reply:
@floreal15 please also report all behavior decided here in the issue body. This way, future maintainance will be easier for developers (they will have the full specification without to serarch in merge requests discussions)